DEPLOY.md

```diff
@@ -103,43 +103,63 @@ Once the `develop` branch for all test projects are reliably passing with the ne
 In the following instructions, "X.Y.Z" is used to denote the version of Cypress being published.
 
 0. Make sure that if there is a new [`cypress-example-kitchensink`](https://github.com/cypress-io/cypress-example-kitchensink/releases) version, the corresponding dependency in [`packages/example`](./packages/example) has been updated to that new version.
-1. Make sure that you have the correct environment variables set up before proceeding.
-    - You'll need Cypress AWS access keys in `aws_credentials_json`, which looks like this:
+1. Make sure you have the correct permissions set up before proceeding:
+    - An AWS account with permission to create AWS access keys for the Cypress CDN.
+    - Permissions for your npm account to publish the `cypress` package.
+2. Make sure that you have the correct environment variables set up before proceeding:
+    - Cypress AWS access key and secret in `aws_credentials_json`, which looks like this:
         ```text
         aws_credentials_json={"bucket":"cdn.cypress.io","folder":"desktop","key":"...","secret":"..."}
         ```
-    - You'll need a [GitHub token](https://github.com/settings/tokens), a [CircleCI token](https://circleci.com/account/api),
+    - A [GitHub token](https://github.com/settings/tokens), a [CircleCI token](https://circleci.com/account/api),
       and a `cypress-io` account-specific [AppVeyor token](https://ci.appveyor.com/api-keys) in `ci_json`:
         ```text
         ci_json={"githubToken":"...","circleToken":"...","appVeyorToken":"..."}
         ```
+    - You'll also need to put the GitHub token under its own variable and get a [ZenHub API token](https://app.zenhub.com/dashboard/tokens) for the `release-automations` step.
+        ```text
+        GITHUB_TOKEN="..."
+        ZENHUB_API_TOKEN="..."
+        ```
+    - The `cypress-bot` GitHub app credentials are also needed. Ask another team member who has done a deploy for those.
     - Tip: Use [as-a](https://github.com/bahmutov/as-a) to manage environment variables for different situations.
-2. Use the `move-binaries` script to move the binaries for `<commit sha>` from `beta` to the `desktop` folder
+3. Use the `move-binaries` script to move the binaries for `<commit sha>` from `beta` to the `desktop` folder
     for `<new target version>`
     ```shell
     yarn move-binaries --sha <commit sha> --version <new target version>
     ```
-3. Publish the new NPM package under the dev tag. The unique link to the package file `cypress.tgz`
-    is the one already tested above. You can publish to the NPM registry straight from the URL:
+4. Publish the new NPM package under the dev tag.
+    - To find the link to the package file `cypress.tgz`:
+        1. In GitHub, go to the latest commit (the one whose sha you used in the last step).
+            ![commit-link](https://user-images.githubusercontent.com/1157043/80608728-33fe6100-8a05-11ea-8b53-375303757b67.png)
```

Comment:
I'm not totally sure how expiry works for these images... will they persist forever even if not referenced in an open issue?

Reply:
As far as I know. We've used this strategy for images for several years and I don't think we've had any issues.